evm-bridging/cadence/transactions/bridge_nfts_to_evm_and_wrap.cdc

```diff
@@ -0,0 +1,189 @@
+import "FungibleToken"
+import "NonFungibleToken"
+import "ViewResolver"
+import "MetadataViews"
+import "FlowToken"
+
+import "ScopedFTProviders"
+
+import "EVM"
+
+import "FlowEVMBridge"
+import "FlowEVMBridgeConfig"
+import "FlowEVMBridgeUtils"
+
+/// Bridges NFTs with provided IDs from Cadence to EVM and wraps them in a wrapper ERC721
+///
+/// @param wrapperERC721Address: EVM address of the wrapper ERC721 NFT
+/// @param nftIDs: Array of IDs of the NFTs to wrap
+///
+transaction(
+    wrapperERC721Address: String,
```

Comment:
This parameter should not be needed anymore once we can get the contract address from new metadata view on-chain